src/modules/users/edit-user/edit-user.js

```diff
@@ -65,7 +65,10 @@
                                     vm.errorMsg = "Please enter a valid Email Id";
                                 }
                             } else if (keys.indexOf("name") !== -1) {
-                                vm.errorMsg = "Name is too short (minimum is 4 characters).";
+                                if(messages.indexOf("is too short (minimum is 4 characters)") !== -1)
```

Comment:
not in favor of doing character/string machine to show an error. Easy way to handle this is with below example.

```
else if (keys.indexOf("name") !== -1) {
    vm.errorMsg = messages;
}
```

Also, I am not in favor of checking this kind of error messages from the API. Before doing an API call we should test this on client side and validate. because I think these are simple validations and can be handled on UI side Easily. @gnehapk thoughts?

Reply:
@umangachapagain Can you please handle this validation from the client side so that API errors do not need to be checked.